### PR TITLE
FF118 RTCRtpTransceiver directions include stopped option

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates [`RTCRtpTransceiver.currentDirection`](https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver/currentDirection) and [`RTCRtpTransceiver.direction`](https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpTransceiver/direction) and friends to note that the way to determine whether a transceiver is stopped is to check these values against the `stopped` string. 

It also spells what these directions actually mean - ie what is a direction, and why you need a preferred vs a current.

... and brings the docs that have been touched in line with current templates.

Related docs work can be tracked in #28845

### CODE REVIEW

File: files/en-us/web/api/rtcpeerconnection/addtrack/index.md [hunk 1]
```diff
@@ -167,7 +167,9 @@ been added to the connection.
 
 This method may return either a new `RTCRtpSender` or, under very specific
```
Comment (bsmth) [inline files/en-us/web/api/rtcpeerconnection/addtrack/index.md:168]: The "not used to transmit data" bit is used twice, do you think we can rephrase this para:

```md
This method returns a new `RTCRtpSender` or an existing instance for reuse.
An `RTCRtpSender` instance is compatible for reuse if it meets the following criteria:
```

File: files/en-us/web/api/rtcpeerconnection/addtrack/index.md [hunk 2]
```diff
@@ -177,25 +179,18 @@ data. Compatible reusable `RTCRtpSender` instances meet these criteria:
...
 
 - The `RTCRtpSender`'s {{domxref("RTCRtpSender.track", "track")}} is set to
   the specified track.
 - The sender's set of associated streams is set to the list of streams passed into
   this method, `stream...`.
-- The associated {{domxref("RTCRtpTransceiver")}} has its
-  `currentDirection` updated to include sending; if its current value is
-  `"recvonly"`, it becomes `"sendrecv"`, and if its current value
-  is `"inactive"`, it becomes `"sendonly"`.
+- The associated {{domxref("RTCRtpTransceiver")}} has its `currentDirection` updated to include sending;
```
Comment (bsmth) [inline files/en-us/web/api/rtcpeerconnection/addtrack/index.md:192]:
```suggestion
- The associated {{domxref("RTCRtpTransceiver")}} has its `currentDirection` updated to indicate it is sending;
```

Comment (bsmth) [reply files/en-us/web/api/rtcpeerconnection/addtrack/index.md:192]: or "to indicate that it is sending", maybe

File: files/en-us/web/api/rtcrtptransceiver/direction/index.md
Comment (bsmth) [inline files/en-us/web/api/rtcrtptransceiver/direction/index.md:1]: super 👍🏻

**Comment by bsmth** [comment]
>It also spells what these directions actually mean - ie what is a direction, and why you need a preferred vs a current.

This is really useful, thank you for adding 👍🏻

**Review by bsmth — APPROVED** [review]
I have some non-blocking comments for your consideration, leaving my +1 - please merge when you are ready

Thank you!

File: files/en-us/web/api/rtcpeerconnection/addtrack/index.md (hunk 1 above)
Comment (hamishwillee) [reply files/en-us/web/api/rtcpeerconnection/addtrack/index.md]: Yes, I like that  a lot more. Thanks.

File: files/en-us/web/api/rtcpeerconnection/addtrack/index.md (hunk 2 above)
Comment (hamishwillee) [reply files/en-us/web/api/rtcpeerconnection/addtrack/index.md:188]: Thanks. Modifying to " "to indicate that it is sending"".

> Also it looks like we can link to /web/api/rtcrtptransceiver/currentdirection/index.md where we describe recvonly, sendrecv etc.

We could link, but I'm not afraid of the duplication, in particular when it was duplicated in the first place.

**Comment by hamishwillee** [comment]
Thanks for the review @bsmth !

**Pull request merged**